### DISCUSSION TITLE
How to add label to an input not part of a form?

### DISCUSSION THREAD

**Discussion by kennyhuynh125** [ant-design/ant-design#30779] [Q&A]
Hi all,

I am wondering what is the best way to add a label to an `<Input />` component that is not part of a form. This is more used for testing purposes with RTL (react-testing-library). For example, I have a component like so:

```
const MyComponent = () => {
  return (
      <Space direction='vertical'>
          <Typography.Text>I want this as my label</Typography.Text>
          <Input />
      </Space>
  )
}
```

It would be nice if I can get the input element with RTL like so:

`screen.getByLabelText(/i want this as my label/i);`

So a few questions:

1. Is there some `<Label />` component that AntD provides to support what I am trying to achieve?
2. I know that the `<Form.Item />` component you can pass in a label prop. Is it recommended that if I want a label to use `<Form.Item />`? My use case is that I have many different inputs that are not necessarily related to each other. For example; I have 3 different inputs but each inputs are independent from each other. Would it still make sense to just wrap each `<Input />` in a `<Form.Item />` so that I can associate a label to each input?

**Comment by shamseerahammedm** [comment]
+1

**Comment by bombillazo** [comment]
I just use Form.Item without the form.

**Reply by Rey-Wang** [reply]
then how to change the layout?

**Comment by natashaamin** [comment]
You can use <ProFormText> https://procomponents.ant.design/en-US/components/field-set#proformtext
